contracts/connection-router/src/state.rs

```diff
@@ -84,31 +89,23 @@ impl<'a> IndexList<ChainEndpoint> for ChainEndpointIndexes<'a> {
 
 #[cw_serde]
 pub struct NewMessage {
-    pub id_on_chain: MessageID, // unique per chain
+    pub uid: CrossChainUid,
```

Comment:
This could probably just be called `id`. Reads a bit cleaner that way.

Reply:
Agree with @cjcobb23

Reply:
yeah, there is going to be some renaming in the follow-ups